integrationtests/tests/fixtures/mod.rs

```diff
@@ -670,19 +690,18 @@ struct ServerTest {
 
 /// Represents a collection of fedimint peer servers
 impl FederationTest {
-    /// Returns the outcome of the last consensus epoch
-    pub fn last_consensus(&self) -> HbbftConsensusOutcome {
-        self.last_consensus.borrow().clone()
-    }
-
-    /// Returns the items that were in the last consensus
-    /// Filters out redundant consensus rounds where the block height doesn't change
-    pub fn last_consensus_items(&self) -> Vec<ConsensusItem> {
-        self.last_consensus()
+    /// Returns the first item with the given module id from the last consensus outcome
+    pub fn find_module_item(&self, id: ModuleInstanceId) -> Option<DynModuleConsensusItem> {
+        self.last_consensus
+            .borrow()
+            .clone()
             .contributions
             .values()
             .flat_map(|items| items.clone())
-            .collect()
+            .find_map(|ci| match ci {
```

Comment:
Might be good to assert there was only one to prevent any accidental non-determinism?

Reply:
Sure I can add that

Reply:
Hmm actually it doesn't really work since you get an item for every peer, plus sometimes you are asserting that no item exists.  I think it's fine because it works similar to how `find` does.